airbyte-ci/connectors/pipelines/pipelines/external_scripts/airbyte_ci_install.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+# !IMPORTANT! This script is used to install the airbyte-ci binary on a Linux or macOS system.
+# Meaning, no external dependencies are allowed as we don't want users to have to run anything
+# other than this script to install the binary.
+
+import os
+import shutil
+import sys
+import tempfile
+import urllib.request
+
+# !IMPORTANT! This constant is inline here instead of being imported from pipelines/consts.py
+# because we don't want to introduce any dependencies on other files in the repository.
+RELEASE_URL = os.getenv("RELEASE_URL", "https://connectors.airbyte.com/files/airbyte-ci/releases")
+
+
+def get_airbyte_os_name():
+    """
+    Returns 'ubuntu' if the system is Linux or 'macos' if the system is macOS.
+    """
+    OS = os.uname().sysname
+    if OS == "Linux":
+        print(f"Linux based system detected.")
```

Comment:
I suggest using `rich` for nicer console output. https://github.com/Textualize/rich

Reply:
I would love to! But thats the downside of this approach. We cannot add an thirdparty dependencies to these files.

If we do then we need the user to run a `pip install` before they run the script.

This is why we have this disclaimer
https://github.com/airbytehq/airbyte/blob/4fca343c1ecbc369031a98da43a1f44a12870ab7/airbyte-ci/connectors/pipelines/pipelines/external_scripts/airbyte_ci_install.py#L4